azure-communication-ui/calling/README.md

```diff
@@ -46,7 +46,14 @@ repositories {
 ```
 Sync project with gradle files. (Android Studio -> File -> Sync Project With Gradle Files)
 
+### Beta Calling SDK and Features
```

Comment:
I do not think we need to share this with Contoso.
They just can go and put any version they would like to test with in the build.gradle.
USE_BETA_CALLING_SDK does not necessary use actual lates released SDK beta.

Reply:
Removed from Readme